ios/brave-ios/Sources/Brave/Frontend/Browser/BrowserViewController/BVC+TabWebDelegate.swift

```diff
@@ -572,4 +613,49 @@ extension BrowserViewController {
       return nil
     }
   }
+
+  func tab(
+    _ tab: Tab,
+    createNewTabWithRequest request: URLRequest,
+    configuration: WKWebViewConfiguration
+  ) -> Tab? {
+    guard !request.isInternalUnprivileged,
+      let navigationURL = request.url,
+      navigationURL.shouldRequestBeOpenedAsPopup()
+    else {
+      print("Denying popup from request: \(request)")
```

Comment:
Logger instead?

Reply:
Technically this is just moved code, I'll switch it to the a logger in my next PR